includes/sanitizers/class-amp-object-sanitizer.php

```diff
@@ -35,7 +34,7 @@ class AMP_Object_Sanitizer extends AMP_Base_Sanitizer {
 	 */
 	public function get_selector_conversion_mapping() {
 		return [
-			Tag::OBJECT => [
```

Comment:
I readded that constant in the latest `main` branch, so this commit can be reverted after an update to the latest toolbox commit.